### PR TITLE
Add number of resolved and open dependent bugzilla bugs.

### PR DESCRIPTION
**Pull request by brendandahl**
Ended up being a bit more difficult since I needed to limit the number of connections to bugzilla.

### CODE REVIEW

**Comment by marco-c** [comment]
Sometimes tracker bugs have a few bugs that remain open even when the feature has shipped. Should we still show X/Y in those cases?

**Comment by brendandahl** [comment]
@digitarald ^

File: engine/throttle.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/**
+ * Limits the concurrency of the `fn`.
+ * @param {Number} concurrency - maxium number of inflight calls to `fn`
+ * @param {Function} fn - function that returns a promise.
+ * @return {Promise}
+ */
+export default function throttle(concurrency, fn) {
```
Comment (digitarald) [inline engine/throttle.js]: Nothing on npm for rate limiting? ;)

**Comment by digitarald** [comment]
@marco-c @brendandahl I think we should obey the tracking bug being closed. This would mean that the remaining bugs weren't blockers (or need to be cleaned up).

Maybe we can throw a warning for that in validation, @brendandahl ?

File: engine/throttle.js (hunk 1 above)
Comment (brendandahl) [inline engine/throttle.js]: There are, two I tried were broken with our node version or didn't use built in promises and were broken. Just found another that seems to be more up to date https://www.npmjs.com/package/bottleneck. I'll give that whirl.

**Pull request merged**